### PR TITLE
feature & fix : added admin job management  and fixed add job form

### PR DESCRIPTION
**Pull request by zakiabdullahi**
1. added admin job management  EDIT & DELETE 
3. added  filtering ,sorting & pagination using Shadcn & Tanstac reacquery for caching data
4.  fixed register form  Responsive

[10xdevs3.webm](https://github.com/user-attachments/assets/a5ef3a8c-4f33-4551-8374-bcf16f671533)

### CODE REVIEW

File: src/app/admin/jobs/[id]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import prisma from '@/config/prisma.config';
+
+import { notFound } from 'next/navigation';
+import PostJobForm from '@/components/job-form';
+
+const UpdateJoPage = async ({ params }: { params: { id: string } }) => {
+  let job;
+
+  try {
+    job = await prisma?.job.findUnique({ where: { id: params.id } });
```
[the remark this answers is not in the document]
Comment (zakiabdullahi) [reply src/app/admin/jobs/[id]/page.tsx:11]: sure  will fix

**Comment by zakiabdullahi** [comment]
fixed  issues   [PR](https://github.com/code100x/job-board/pull/224)

**Comment by zakiabdullahi** [comment]
@VineeTagarwaL-code

**Comment by zakiabdullahi** [comment]
@VineeTagarwaL-code

**Comment by VineeTagarwaL-code** [comment]
fix conflicts, and remove un-unessecary dependencies or leverage it properly

**Comment by zakiabdullahi** [comment]
fixed it

**Comment by VineeTagarwaL-code** [comment]
a better pr is opened, will go ahead with that rather than  this one, sorry for this but please lookout for other issues will happily review and merge them

**Pull request closed without merging**